### PR TITLE
4 Version Bumps

### PR DESCRIPTION
**Pull request by bassmadrigal**
network/discord: Version bump to 0.0.32
games/SpaceCadetPinball: Version bump to 2.1.0
python/python3-tox: Version bump to 4.11.3
libraries/gtk-sharp3: Version bump to 3.22.2 and new upstream

### CODE REVIEW

**Comment by aclemons** [comment]
@sbo-bot: build x86_64 network/discord

**Comment by aclemons** [comment]
@sbo-bot: build python/python3-tox

**Comment by aclemons** [comment]
@sbo-bot: build games/SpaceCadetPinball

**Comment by aclemons** [comment]
@sbo-bot: build libraries/gtk-sharp3

**Comment by aclemons** [comment]
https://slackbuilds.org/advsearch.php?q=gtk-sharp3&stype=revdep

```
gtk-sharp3
└ gbrainy
```

**Comment by aclemons** [comment]
https://slackbuilds.org/advsearch.php?q=python3-tox&stype=revdep

```
python3-tox
├ python3-cinemagoer
│ └ sickchill
└ python3-solidpython
  └ python3-gds3xtrude
```

File: libraries/gtk-sharp3/gtk-sharp3.SlackBuild [hunk 1]
```diff
@@ -80,26 +80,36 @@ find -L . \
...
-
-make
-make install DESTDIR=$PKG
+# Correct outdated references
+sed -i 's|gdk_api_includes|gio_api_includes|' \
+  Source/{gdk,gio,gtk}/generated/meson.build \
+  Source/sample/valtest/generated/meson.build
+
+# Mono packages use /usr/lib regardless of arch
+patch -p 1 < $CWD/gtk-sharp3-nolibdir.patch
```
Comment (aclemons) [inline libraries/gtk-sharp3/gtk-sharp3.SlackBuild:89]: @bassmadrigal Looks like you missed staging this patch into git.

**Comment by aclemons** [comment]
@sbo-bot: build python/python3-cinemagoer

**Comment by aclemons** [comment]
@sbo-bot: build multimedia/sickchill

**Comment by aclemons** [comment]
@sbo-bot: build python/python3-solidpython

**Comment by aclemons** [comment]
@sbo-bot: build graphics/python3-gds3xtrude

File: libraries/gtk-sharp3/gtk-sharp3.SlackBuild (hunk 1 above)
Comment (bassmadrigal) [reply libraries/gtk-sharp3/gtk-sharp3.SlackBuild:89]: Just added. Thanks for the catch!

**Comment by aclemons** [comment]
@sbo-bot: build libraries/gtk-sharp3

**Comment by aclemons** [comment]
@sbo-bot: build games/gbrainy

**Review by aclemons — APPROVED** [review]
this LGTM - thanks @bassmadrigal

**Pull request merged**